variables.tf

```diff
@@ -498,9 +498,28 @@ variable "redis_subnet_tag_filter" {
 # -------------------------------------------------------------------------------------------------
 variable "rds_enabled" {
   description = "Set to false to prevent the module from creating any rds resources"
+  type        = bool
   default     = false
 }
 
+variable "rds_enable_s3_dump" {
+  description = "Set to false to prevent the module from creating any rds dump resources"
```

Comment:
I would formulate this a bit different: _Set to true to allow the module to create RDS dump resources._
The current wording implies that the default is `true`.